v63002/u_inref/gtm8694.csh

```diff
@@ -37,6 +37,7 @@ TRIGGER_MOD
 EOF
 # Not including CENABLE in this list because of logistics reasons
 echo "BREAK ZBREAK ZCMDLINE ZEDIT ZSYSTEM PIPE_OPEN DIRECT_MODE DSE TRIGGER_MOD">restrictlist.txt
+echo "breakfn zbreakfn zcmdlnefn zeditfn zsystemfn pipefn trigmodfn">fnlist.txt
```

Comment:
Why was this change needed?

Reply:
This allows me to randomly select a function to execute for the last case

Reply:
Got it.